lib/web_of_science/identifiers.rb

```diff
@@ -38,10 +40,30 @@ def doi
     end
 
     # @return [String|nil]
+    def doi_uri
+      "#{Settings.DOI.BASE_URI}#{doi}" if doi.present?
+    end
+
+    # @return [String|nil]
```

Comment:
YARD uses commas for types, not pipes, like `[String, nil]`:
http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md#Declaring_Types

Reply:
Fixed